stream-payment-order/payment-order-core/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.backbase.stream</groupId>
+        <artifactId>stream-payment-order</artifactId>
+        <version>3.2.0</version>
+    </parent>
+
+    <artifactId>payment-order-core</artifactId>
+
+    <packaging>jar</packaging>
+    <name>Stream :: Payment Order Core</name>
+
+    <properties>
+        <checkstyle.disable.checks>true</checkstyle.disable.checks>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+            <version>2.13.2</version>
```

Comment:
have you checked if this version isn't already managed by any BOM somewhere?

Reply:
Actually, I'm no longer using that lib. Going to remove this. Thanks.